source/developers/transform-data-with-object-lambda.rst

```diff
@@ -0,0 +1,242 @@
+.. _developers-object-lambda:
+
+=================================
+Transform Data with Object Lambda
+=================================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+The following documentation covers transforming the contents of an object with MinIO's Object Lambda.
+Uses for Object Lambda include redacting personally identifable information (PII) and enriching data with information from other sources.
+
+This documentation assumes the following:
+
+- An existing :ref:`MinIO <minio-installation>` deployment
+- Working Python (3.8+) and Golang (??) development environments
+- :doc:`The MinIO Go SDK </developers/go/minio-go>` installed
+  
+-------
+Example
+-------
+
+This example creates a Lambda handler in Python, invokes it from a Go function, and retrieves the transformed object with ``curl``.
+
+
+Create a Lambda Handler
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The sample handler, written in Python, retrieves the target object using a `presigned URL <https://min.io/docs/minio/linux/developers/go/API.html#presigned-operations>`__. It then transforms the object's contents and returns the new text to the caller.
+It uses `the Flask web framework <https://flask.palletsprojects.com/en/2.2.x/>`__ and Python 3.8+. 
+
+The following command installs Flask and other needed dependancies:
+
+.. code-block:: shell
+   :class: copyable
+
+   pip install flask requests
+
+The handler uses information from the MinIO event context to get the original object, along with the tokens needed to validate the response when the transformed object is returned.
+In the example below, ``object_context`` contains several values from the ``getObjectContext`` property of the JSON request payload:
+
+* ``inputS3Url`` is a presigned URL for the original object. A `presigned URL <https://min.io/docs/minio/linux/developers/go/API.html#presigned-operations>`__ allows the function to access an object without the MinIO credentials usually required. 
+* ``outputRoute`` and ``outputToken`` are tokens added to the response headers to validate the response and return it to the correct user.
```

Comment:
More vagueness, particularly about `outputRoute`. I presume that is how we know who to send the response back to, what additional can we say about that?

Reply:
Maybe
```
* MinIO Lambda adds `outputRoute`` and ``outputToken`` to the response headers to validate the response and return it to the correct user.
```